src/components/related-overlay/related-overlay.tsx

```diff
@@ -42,7 +42,11 @@ const RelatedOverlay = connect(mapStateToProps)(({relatedManager, isPaused, isPl
     return <></>;
   } else if (!isPlaybackEnded) {
     setIsVisible(
-      isPaused && relatedManager.showOnPlaybackPaused && sizeBreakpoint !== PLAYER_SIZE.EXTRA_SMALL && sizeBreakpoint !== PLAYER_SIZE.SMALL
+      isPaused &&
+        relatedManager.showOnPlaybackPaused &&
+        sizeBreakpoint !== PLAYER_SIZE.TINY &&
+        sizeBreakpoint !== PLAYER_SIZE.EXTRA_SMALL &&
+        sizeBreakpoint !== PLAYER_SIZE.SMALL
```

Comment:
it can be shorter by
 `&& ![PLAYER_SIZE.TINY, PLAYER_SIZE.EXTRA_SMALL, PLAYER_SIZE.EXTRA_SMALL].includes(sizeBreakpoint)`